css/blur-no-alt-editor.css

```diff
@@ -1,32 +1,32 @@
 .block-editor-writing-flow img:is([alt=""],[alt*='.jpg'],[alt*='.png'],[alt*='.gif']) {
-	clip-path: inset(0);
-	filter: blur(8px);
-	outline: 10px solid red;
-	outline-offset: -10px;
-	transition: filter .125s, outline .125s;
+	clip-path: var(--blur-no-alt--clip-path, inset(0)) !important;
+	filter: var(--blur-no-alt--filter, blur(8px)) !important;
+	outline: var(--blur-no-alt--outline, 10px solid red) !important;
+	outline-offset: var(--blur-no-alt--outline, 10px solid red) !important
+	transition: var(--blur-no-alt--transition, filter .125s, outline .125s) !important;
```

Comment:
I love CSS variables. But what are we gaining by using them here, and this way throughout? Or as I have been known to ask, WAYRTTDH? (What are you really trying to do here?)

Reply:
Proposal: I think it would be great to add some very short examples of CSS that takes advantage of the custom properties in the README so this is documented. I'd be happy to add that as part of this PR.